components/form.css

```diff
@@ -0,0 +1,82 @@
+.form{
+    width: 100%;
+}
+
+.label{
+    font-size: 1.25rem;
+    letter-spacing: initial;
+}
+
+.input{
+    margin: 10px auto;
+    border-radius: 4px;
+    outline: none;
+    border: 1px solid var(--gray-700);
+    padding: 10px 1rem;
+    width: 100%;
+    font-size: 14px;
+}
+
+.input_container{
+    margin-bottom: 2rem;
+}
+
+.field_required::after{
+    content: "*";
+    color: var(--red-400);
+}
+
+.form input:focus {
+	outline: 0;
+	border-color: var(--gray-100);
+}
+
+.input_container select{
+    padding: 8px 1rem;
+    font-size: 1rem;
+    width: 20%;
+    margin-left: 8px;
+}
+
+.form_valid .msg_success,
+.form_valid .msg_error{
+    margin-bottom: 8px;
+}
+
+.form_valid .msg_success i.fa-check-circle,
+.form_valid .msg_success{
+    color: var(--form-success); 
+}
+
+.form_valid .msg_error i.fa-exclamation-circle,
+.form_valid .msg_error{
+    color: var(--form-error);
+}
+
+.form_valid .msg_error,
+.form_valid .msg_success{
+    display: none;
+}
+
+.form_valid.success input:valid{
+    border-color: var(--form-success);
+}
+
+.form_valid.success input:valid ~ .msg_success{
+    display: block;
+}
+
+.form_valid.success input:invalid{
+    border-color: var(--form-error);
+}
+
+.form_valid.success input:invalid ~ .msg_error{
+    display: block;
+}
+
+[type="checkbox"] {
+    outline: none;
+    margin-right: 8px;
+    margin-bottom: .75rem;
```

Comment:
can change this to px.

Reply:
I have used pixel for units less than 10px.